dRep.spec

```diff
@@ -0,0 +1,16 @@
+/*
+A KBase module: dRep
+*/
+
+module dRep {
+    typedef structure {
+        string report_name;
+        string report_ref;
+    } ReportResults;
+
+    /*
+        This example function accepts any number of parameters and returns results in a KBaseReport
+    */
+    funcdef dereplicate(mapping<string,UnspecifiedObject> params) returns (ReportResults output) authentication required;
```

Comment:
Add a full struct type for the params so people can see your interface. It's also helpful to make comments for the params struct to document each field.